packages/mobile/src/identity/contactMapping.test.ts

```diff
@@ -90,4 +108,90 @@ describe('Fetch Addresses Saga', () => {
       )
       .run()
   })
+
+  it('requires SecureSend with partial verification when a new adddress is added and last 4 digits are unique', async () => {
+    const contractKit = await getContractKitOutsideGenerator()
```

Comment:
Why is this here? Seems like you don't use it and pass in a mockContractKit instead, which seems right to me

Reply:
I think I need it. I pass in `mockAttestationWrapper` but still use `contractKit` in the provide method.